plugins/modules/postgresql_owner.py

```diff
@@ -292,7 +366,11 @@ def __is_owner(self):
                      "WHERE d.datname = %(obj_name)s "
                      "AND r.rolname = %(role)s")
 
-        elif self.obj_type == 'function':
+        elif self.obj_type in ('aggregate', 'function', 'routine', 'procedure'):
+            if self.obj_type == 'routine' and self.pg_version < 110000:
+                raise Error("PostgreSQL version must be >= 11 for obj_type=routine. Exit")
```

Comment:
```suggestion
                self.module.fail_json(msg="PostgreSQL version must be >= 11 for obj_type=routine.")
```
this is a standard way of handling such stuff, i.e. all interactions with a user are done through the `module` object . Relevant for all similar cases. I would also suggest removing `Exit` from the messages.

Reply:
Good advice, fixed.